### ISSUE TITLE
Password reset not sending email?

### ISSUE THREAD

**Issue by fernandodiacenco** [zammad/zammad#773]
Hello,

I installed and configured Zammad 1.3 on Ubuntu 16.04.1

The user can create tickets by email, receives an email when is answered on Zammad, everything is working correctly so far

But when the user tries to reset password (when trying to access the menu where it can see opened tickets) nothing seems to happen, no email is sent, but no error message is shown. On the user interface it appears as if nothing is wrong

We tried a few different configurations without success

Is there any specific configuration to be set?

Thanks!

**Comment by thorsteneckel** [comment]
Hi @fernandodiacenco - have you checked your logfile? Could you please provide the log when requesting a password reset? There should be no other configuration for using this functionality.

**Comment by fernandodiacenco** [comment]
The logs didn't show any error, only infos that the notification was sent to the specified address

I tried using an internal email relay and it worked, so problem solved

But, when the email for password reset arrives, it comes as (for example) tasks.company.com instead of tasks.company.com:3000, manually adding the port works

I am looking for a way to redirect to port 3000 from 80 to bypass the "welcome to nginx" page, my google-fu failed me so far, can you provide guidance?

Thanks.

**Comment by monotek** [comment]
Maybe notifications are send via sendmail. 
If so, what says your mail.log?

**Comment by fernandodiacenco** [comment]
Interesting, I tried a fresh install with both sendmail and postfix, and received the same error:

> Feb 20 12:36:57 tarefas postfix/pickup[2007]: 46D77361854: uid=107 from=<noreply@tarefas.confitec.com.br>
> Feb 20 12:36:57 tarefas postfix/cleanup[2185]: 46D77361854: message-id=<notification.20170220153657.2.6.823829@tarefas.confitec.com.br>
> Feb 20 12:36:57 tarefas postfix/qmgr[1151]: 46D77361854: from=<noreply@tarefas.confitec.com.br>, size=5318, nrcpt=1 (queue active)
> Feb 20 12:36:57 tarefas postfix/smtp[2187]: 46D77361854: host mx.a.locaweb.com.br[186.202.4.42] said: 450 4.1.8 <noreply@tarefas.confitec.com.br>: Sender address rejected: Domain not found (in reply to MAIL FROM command)

The dns of the server is on our local dns, and the server itself can resolve to any hostname inside or outside our network

Searching the net, one guy who had a similar problem fixed by adding the server to the hosts file, I tried that as well without luck, here is my cat /etc/hosts

> 127.0.0.1       localhost tarefas tarefas.confitec.com.br
> 127.0.1.1       tarefas tarefas.confitec.com.br
> The following lines are desirable for IPv6 capable hosts
> ::1     localhost ip6-localhost ip6-loopback
> ff02::1 ip6-allnodes
> ff02::2 ip6-allrouters

tried dpkg-reconfigure postfix too, no luck

It might be a silly configuration on the mta that I'm not familiar with, can anybody shed some light?

Also, it is easy to redirect nginx from port 3000 to 80?

Thanks

**Comment by monotek** [comment]
mx.a.locaweb.com.br cant find your tarefas.confitec.com.br domain and therefore will not accept mails from your server. You need external DNS to get it work. Maybe some postfix howto will help: https://help.ubuntu.com/community/PostfixBasicSetupHowto

Our nginx config already listens on port 80: https://github.com/zammad/zammad/blob/develop/contrib/nginx/zammad.conf

**Comment by fernandodiacenco** [comment]
Ok so i tried reinstalling again and running the wizard to try to understand what went wrong on the first time, since the config seems pretty straightforward I might be missing something

And indeed, the first time i tried outgoing smtp settings first before trying local mta, and it gave me errors, so i jumped to mta and forgot about it

But it seems to make more sense to use this option, so on it when i try the email with tarefas@confitec.com.br it complains about needing fully qualified domain name

BUT when i try the same config with tarefas@confitec.com (without .br) it complains about failed authentication (i do not own confitec.com, only confitec.com.br)

(I attached screenshots illustrating the error)

I put the server fqdn on zammad.conf (server_name directive) and on the server hosts file

What i am missing?

> Our nginx config already listens on port 80: https://github.com/zammad/zammad/blob/develop/contrib/nginx/zammad.conf

I don't understand why when I try the fqdn, what I get is the "welcome to nginx" page instead of the zammad front page, I must be missing something, can the two problems be related somehow? (fqdn issues)

Thanks!

![01](https://cloud.githubusercontent.com/assets/2643364/23146084/5f1a6af2-f7b2-11e6-940d-88e1e5b6e353.png)

---

![02](https://cloud.githubusercontent.com/assets/2643364/23146085/5f1aa850-f7b2-11e6-913f-56225a3873ab.png)

**Comment by monotek** [comment]
You have to configure the right "Notification Sender" in "Channels / Email / Settings" too.
 
Fqdn is a completely different topic and has nothing to do with sending mails. If you want to use a domain in nginx you have to configure it in the server_name variable. This only works if you have a DNS entry for this domain or put the domain in the hosts file of the client which is accessing Zammad via browser.

**Comment by fernandodiacenco** [comment]
Well, I managed to solve one issue and have a workaround on the other

On the port 80 thing, I noticed that even after a fresh install, on /etc/nginx/sites-availble there was two files, default and zammad, both listening on 80, renaming/removing default one, defining server_name on zammad.conf and restarting nginx fixed the issue.

The email notifications absolutely refused to work with the account provided, but when I used one decommissioned account hosted on gmail for business (smtp.gmail.com port 465), it worked. So I'm not sure why it didn't work with my current email provider but whatever, its working now.

Also I congratulate you guys for the translation tool inside Zammad, what and amazing idea, I will surely contribute to it.

Thanks for your help!
